pandas/io/tests/test_common.py

```diff
@@ -4,10 +4,22 @@
 from pandas.compat import StringIO
 import os
 
+import nose
 import pandas.util.testing as tm
 
 from pandas.io import common
 
+try:
+    import pathlib
+    PATHLIB_INSTALLED = True
```

Comment:
make these into functions that raise ``nose.SkipTest`` (e.g. put them in this module, but see how this is done in ``utils.testing``, e.g. the ``_skip_if......``